spec/models/note_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe Note, :type => :model do
```

Comment:
Hash rocket here ^

```suggestion
RSpec.describe Note, type: :model do
```

Just pointing out as once you start contributing to JR plus you will get picked up for all hash rockest 🙂